.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,15 @@
+name: Autobump version
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  autobump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: bump
+        run: ./bump_version.sh && git push -f origin master || echo 'No action needed.'
```

Comment:
Should be origin main

Reply:
Oops, fixed.

Reply:
O would also avoid force pushing, just in case...

Reply:
If we amend then we have to force push. I don't know how to do a separate commit from this action.

Reply:
This wouldn't work with `--amend`.